packages/deployer/deploy/deploy.ts

```diff
@@ -209,9 +127,26 @@ export default async function deploymentFunc (this: DeployFunction, hre: Hardhat
   }
   const hub = new ethers.Contract(relayHub.address, relayHub.abi, ethers.provider.getSigner())
 
-  if (relayHub.newlyDeployed) {
-    console.log('Adding stake token', stakingTokenAddress, 'stake=', stakingTokenValue)
-    await hub.setMinimumStakes([stakingTokenAddress], [stakingTokenValueParsed.toString()])
+  const tokens: Array<{ token: string, minimumStake: string } | null> = await Promise.all(
```

Comment:
This is a confusing chained call, is it ok if I ask you to break it up? 
Please define a function `checkMinimumStakeSetOnHub(): Promise<***>`
Define a `const tokensPromises: Array<Promise<***>>`
And then do the `Promise.all` ?